applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -188,65 +166,88 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay);
 
 /**
- * @brief	Increase volume by one step
+ * @brief	Set pointer to connection, used by CIS.
  *
- * @return	0 for success,
- *		-ENXIO if the feature is disabled,
- *		error otherwise
+ * @param	conn	The connection pointer.
  */
-int le_audio_volume_up(void);
+void le_audio_conn_set(struct bt_conn *conn);
 
 /**
- * @brief	Decrease volume by one step
+ * @brief	Set periodic advertising sync.
  *
- * @return	0 for success,
- *		-ENXIO if the feature is disabled,
- *		error otherwise
+ * @param	pa_sync		Pointer to the periodic advertising sync.
+ * @param	broadcast_id	Broadcast ID of the periodic advertising.
+ *
+ * @return	0 for success, (negative) error code otherwise.
  */
-int le_audio_volume_down(void);
+int le_audio_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_id);
 
 /**
- * @brief	Mute volume
+ * @brief	Notify about disconnected connection, used by CIS.
  *
- * @return	0 for success,
- *		-ENXIO if the feature is disabled,
- *		error otherwise
+ * @param	conn	The connection pointer.
+ */
+void le_audio_conn_disconnected(struct bt_conn *conn);
+
+/**
+ * @brief	Set pointer to extended advertisement with periodic adv configured, used by BIS.
+ *
+ * @note	Will also start the broadcast_source.
+ *
+ * @param	ext_adv	Pointer to the extended advertisement.
+ *
+ * @return	0 for success, (negative) error code otherwise.
  */
-int le_audio_volume_mute(void);
+int le_audio_ext_adv_set(struct bt_le_ext_adv *ext_adv);
 
 /**
- * @brief	Either resume or pause the Bluetooth LE Audio stream,
- *		depending on the current state of the stream
+ * @brief	Get bt_data containing the data to advertise.
  *
- * @return	0 for success, error otherwise
+ * @param	adv		Pointer to the pointer of bt_data to advertise.
```

Comment:
```suggestion
 * @param	adv		Pointer to the pointer of @p bt_data to advertise.
```

Reply:
bt_data is not a parameter in this function

Reply:
It is the type of struct that we pass the double pointer to